sys/parsoid.js

```diff
@@ -219,7 +219,13 @@ class ParsoidService {
         ]);
     }
 
-    getNGBucketURI(rp, format, tid) {
+    getLatestBucketURI(domain, title, revision, tid) {
+        return new URI([
+            domain, 'sys', 'key_value', 'parsoid', `${title}:${revision}:${tid}`
```

Comment:
If this is the latest bucket, why are we storing the title together with the revision and tid? it should just be the title

Reply:
This is wrong, you are right. THis is not even used in this PR. Shall remove.